perses/rjmc/geometry.py

```diff
@@ -616,9 +632,9 @@ def _internal_to_cartesian(self, bond_position, angle_position, torsion_position
         Calculate the cartesian coordinates given the internal, as well as abs(detJ)
```

Comment:
this requires a modification to coordinate_numba, otherwise there are compatibility errors in the unit system of choice

Reply:
i can modify it, but it may cause other tests that use coordinate numba to fail

Reply:
Are you sure it requires modifying `coordinate_numba`? My understanding was that all the methods in there assume quantities are unitless `np.float64` types in a compatible unit system. 

The reason for using `quantity.value_in_unit_system(unit.md_unit_system)` rather than arbitrarily dividing by some units is to *ensure* everything ends up in a self-consistent unit system.

Reply:
```python
~/github/perses/perses/rjmc/geometry.py in _internal_to_cartesian(self, bond_position, angle_position, torsion_position, r, theta, phi)
    614         """
    615         from perses.rjmc import coordinate_numba
--> 616         bond_position = bond_position.value_in_unit(units.md_unit_system).astype(np.float64)
    617         angle_position = angle_position.value_in_unit(units.md_unit_system).astype(np.float64)
    618         torsion_position = torsion_position.value_in_unit(units.md_unit_system).astype(np.float64)

~/anaconda3/lib/python3.7/site-packages/simtk/unit/quantity.py in value_in_unit(self, unit)
    619         Returns underlying value, in the specified units.
    620         """
--> 621         val = self.in_units_of(unit)
    622         if is_quantity(val):
    623             return val._value
```

Reply:
so its not ``` python coordinate numba ```

Reply:
Did you mean to paste a complete stack trace? There isn't an error report in this partial trace.

Reply:
```
~/github/perses/perses/rjmc/geometry.py in _internal_to_cartesian(self, bond_position, angle_position, torsion_position, r, theta, phi)
    614         """
    615         from perses.rjmc import coordinate_numba
--> 616         bond_position = bond_position.value_in_unit(units.md_unit_system).astype(np.float64)
    617         angle_position = angle_position.value_in_unit(units.md_unit_system).astype(np.float64)
    618         torsion_position = torsion_position.value_in_unit(units.md_unit_system).astype(np.float64)

~/anaconda3/lib/python3.7/site-packages/simtk/unit/quantity.py in value_in_unit(self, unit)
    619         Returns underlying value, in the specified units.
    620         """
--> 621         val = self.in_units_of(unit)
    622         if is_quantity(val):
    623             return val._value

~/anaconda3/lib/python3.7/site-packages/simtk/unit/quantity.py in in_units_of(self, other_unit)
    655         """
    656         if not self.unit.is_compatible(other_unit):
--> 657             raise TypeError('Unit "%s" is not compatible with Unit "%s".' % (self.unit, other_unit))
    658         f = self.unit.conversion_factor_to(other_unit)
    659         return self._change_units_with_factor(other_unit, f)

TypeError: Unit "nanometer" is not compatible with Unit "UnitSystem([nanometer, dalton, picosecond, elementary charge, kelvin, mole, radian])".
```
sorry

Reply:
i'll try to figure out where this is coming from